components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -799,9 +799,25 @@ function ImageBuildView(props: ImageBuildViewProps) {
                 <WorkspaceLogs logsEmitter={logsEmitter} errorMessage={props.error?.message} />
             </Suspense>
             {!!props.onStartWithDefaultImage && (
-                <button className="mt-6 secondary" onClick={props.onStartWithDefaultImage}>
-                    Continue with Default Image
-                </button>
+                <>
+                    <div className="mt-6 w-11/12 lg:w-3/5">
+                        <p className="text-center text-gray-400 dark:text-gray-500">
+                            <span className="font-bold">Did you know?</span> You can use <code>gp rebuild</code> to
+                            validate and debug your custom workspace image before pushing &nbsp;&middot;&nbsp;
```

Comment:
**suggestion**: We could trim some text here to make it easier to parse and read. Haven't checked if linting fails with the following code suggestions. 😅 

```suggestion
                            💡 You can use the <code>gp rebuild</code> command to
                            validate the configuration and rebuild the workspace. 
```

We could also skip the validation part as this could be self-explanatory, making the message even easier to read and more specific where you can use the `rebuild` command.

```suggestion
                            💡 You can use the <code>gp rebuild</code> command to
                             rebuild the workspace from the editor terminal. 
```

Reply:
Thanks for the inspiration! I've updated the message, check if you like it or have more suggestions 🙏

Reply:
Looks shippable, @andreafalzetti! Here are background thoughts on the [copy suggestion above](https://github.com/gitpod-io/gitpod/pull/15774#discussion_r1072095821), but the current state looks good for this iteration, and we can update the copy later.

1. The _rebuild_ term was referring to the image build of the container image.
2. The _workspace_ term  was trying to communicate this is about your workspace and nothing scary like images, containers, etc which may be strange to some users.
3. The _command_ term reference was trying to make it clear this is an actual command you have to type in a terminal or somewhere to run it.
4. The _debug_ term was removed to make it less scary to users.
5. The _editor_ term was used to clarify this action can happen inside an editor. Could be removed as this can happen in a terminal locally if you connect with SSH.
6. The _terminal_ term was used to make it clear you need to type this command in a terminal and it's not something you find on the editor UI.

Reply:
@gtsiolis for context, we'll also enable users calling `gp rebuild` from the UI (vscode only for now), see [context](https://github.com/gitpod-io/gitpod/pull/15735), but it feels very hard compress all these info in a line. For now we can go with the "terminal" option and later once we create more generic docs on gitpod.io/docs, we can rephrase this and link there to be a generic "rebuild" feature